CHANGES.md

```diff
@@ -1 +1,48 @@
-## DC/OS 1.12.0
\ No newline at end of file
+## DC/OS 1.12.0
+
+```
+* For any significant improvement to DC/OS add an entry to Fixed and Improved section.
+* For Security updates, please call out in Security updates section.
+* Add to the top of the existing list.
+* External Projects like Mesos and Marathon shall provide a link to their published changelogs.
+
+Format of the entries must be.
+
+* Entry with no-newlines. (DCOS_OSS_JIRA)
+<new-line>
+* Entry two with no-newlines. (DCOS_OSS_JIRA_2)
+```
+
+
+### Fixed and improved
+
+* Windows-based pkgpanda builds (DCOS_OSS-1899)
+
+* Upgrade REX-Ray to v0.11.2. (DCOS_OSS-2245)
```

Comment:
- Can this go into a 'Notable changes' section?

- Upgraded to ...

Reply:
We should give guidance on what qualifies as Notable change.

I don't know why this qualifies as notable change. Could you please share why this should be mentioned in "Notable changes" section.  I am assuming you already thought about this.

Reply:
Great point. My rough thinking here is that if the content of the item does not directly communicate what has been "improved" or "fixed" then it's not suitable for the "Fixed and improved" section, but should go into a more neutral section.

Or, in other words, for every point in "Fixed and improved" I want to know what about the change has fixed or improved something, and it's not really up for interpretation.

These items should mainly be bugfixes, performance improvements, resilience improvements, UX improvements, etc.

The following are examples for notable changes (changes of significance, but the kind of the significance is not communicated, and depends on the point of view):
"Platform: the Java Developer Kit was upgraded to JDK 8u152."
"DC/OS is now compatible with Docker version 17.05.0"
"DC/OS is now compatible with RHEL 7.4."

This is brainstorm. Can surely be reduced to something more concise. Taking note of this for a 'guidelines' document.